horizon/middleware_manager.py

```diff
@@ -0,0 +1,144 @@
+import asyncio
+from copy import deepcopy
+import json
+from datetime import datetime
+
+from app.database import CREDENTIALS
+from horizon.middleware_api import get_players_online, authenticate_async, authenticate, get_active_games, get_recent_stats, get_recent_game_history
```

Comment:
`get_recent_stats` and `get_recent_game_history` are unused imports. Recommend wrapping these imports in `()`.